client/my-sites/people/subscribers-team/index.tsx

```diff
@@ -36,7 +36,7 @@ function SubscribersTeam( props: Props ) {
 
 	// fetching data config
 	const followersFetchOptions = { search };
-	const defaultTeamFetchOptions = { include_viewers: true };
+	const defaultTeamFetchOptions = { include_viewers: true, total_users: ! search };
```

Comment:
We don't need to call `count_users` if there's a search for this component.

This is a micro optimization as `count_users` can potentially be CPU intensive, albeit in rare cases: https://core.trac.wordpress.org/ticket/29785#comment:19